src/java.ts

```diff
@@ -76,7 +76,8 @@ export async function serverSupportsJREProvisioning(
   parameters: ScannerProperties,
 ): Promise<boolean> {
   if (parameters[ScannerProperty.SonarScannerInternalIsSonarCloud] === 'true') {
-    return true;
+    //TODO: return to true once SC has the new provisioning mechanism in place
+    return false;
```

Comment:
this was to force the scanner to run the scanner-cli for SC. this needs to be reverted before we merge)